tools/asm_processor/build.py

```diff
@@ -1,64 +1,114 @@
 #!/usr/bin/env python3
```

Comment:
Now that we have two files from asm_processor, is it worth a subrepo?

Reply:
On that note it would be nice to include the LICENSE file from asm-processor.